source/includes/steps-create-index.yaml

```diff
@@ -37,29 +60,28 @@ content: |
 
       * - Build index in the background
 
-        - If checked, ensure that the MongoDB deployment remains
-          available during the index build operation.
+        - Ensure that the MongoDB deployment remains available during
+          the index build operation.
 
         - :manual:`Background Construction </core/index-creation/index.html#background-construction>`
 
       * - Create unique index
 
-        - If checked, ensure that the indexed fields do not
-          store duplicate values.
+        - Ensure that the indexed fields do not store duplicate values.
 
         - :manual:`Unique Indexes </core/index-unique>`
 
-      * - Create :abbr:`TTL (Time to Live)`
+      * - Create a :abbr:`TTL (Time to Live)` index
 
-        - If checked, automatically delete documents after a
-          specified number of seconds since the indexed field value.
+        - Automatically delete documents after a specified number of
```

Comment:
**nitpick:** Consider not starting some of these with adverbs for the sake of [parallelism](https://www.grammarly.com/blog/parallelism/).